pycernan/avro/base_client.py

```diff
@@ -30,12 +34,18 @@ def _recv_exact(self, sock, n_bytes):
 
     def _send(self, payload_id, sync, payload):
         with self.pool.connection() as sock:
+            metrics.publish_count.inc()
```

Comment:
We should publish metrics after sending the data, not before

Reply:
No, this is the number of attempted publishes. We track failures separately.